CGPs/0020.md

```diff
@@ -0,0 +1,36 @@
+# CGP [0020]: Update randomnessBlockRetentionWindow to Extend Attestation Expiration Duration
+
+- Date: 2021-01-29
+- Author(s): @anna-carroll
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
+- Discussions To: [TODO: create and add discussion link]
+
+## Overview
+
+This proposal will extend the expiration of a requested attestation from 1 HOUR -> 1 DAY, improving upon CGP 0015 which previously attempted to do so.
```

Comment:
I would maybe add a word like `effectively` or `indirectly` in here

Reply:
good call 😄 https://github.com/celo-org/celo-proposals/pull/155/commits/027e9e46a1cd0b89e060f7a270932ab1f71166f2